packages/amplify-cli/src/init-steps/preInitSetup.ts

```diff
@@ -3,11 +3,24 @@ import { execSync } from 'child_process';
 import * as fs from 'fs-extra';
 import * as url from 'url';
 import { generateLocalEnvInfoFile } from './s9-onSuccess';
+import { printer, prompter } from '@aws-amplify/amplify-prompts';
+import { isNewProject } from './s0-analyzeProject';
+
+export const preInitSetup = (isHeadless) => {
```

Comment:
Since this is a higher-order function now, it should probably be named `getPreInitSetup`

Reply:
Good point. Done!

Reply:
[08e896a](https://github.com/aws-amplify/amplify-cli/pull/13849/commits/08e896acd6a3562f1c03fce975b33c92f7fc9891)